jf_agent/git/github.py

```diff
@@ -319,7 +313,7 @@ def get_commits_for_included_branches(
                             )
 
                 except Exception as e:
-                    print(f':WARN: Got exception for branch {branch}: {e}. Skipping...')
+                    logger.info(f':WARN: Got exception for branch {branch}: {e}. Skipping...')
```

Comment:
Q: Should this (and the line below) be a warning?

Reply:
*code on line 432

Reply:
Yup, this is a find and replace error I think